inventories/seapath_cluster_definition_example.yml

```diff
@@ -31,6 +32,18 @@ all:
                             cluster_ip_addr: "192.168.55.1"
                             br_rstp_priority: 12288
                             brBRIDGE1_ext: eno1234 #physical nic to use with BRIDGE1 (see the ovs topology inventory)
+                            custom_network:
+                              - 01-enp24s0_sriov:
```

Comment:
This change is not relative with this commit.

Reply:
indeed, I will split the commit